gulpfile.js

```diff
@@ -219,6 +219,16 @@ const mappings = {
     'dir': 'powerbiembedded/lib',
     'source': 'arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json'
   },
+  'rdbms.mysql': {
+    'packageName': 'azure-arm-rdbms',
```

Comment:
Oh I see. you want to generate a single SDK entrypoint for both these services. In that case, this is correct.

---
In reply to: [118064872](https://github.com/Azure/azure-sdk-for-node/pull/2173#discussion_r118064872) [](ancestors = 118064872)